weave/index/index_pandas.py

```diff
@@ -134,8 +138,11 @@ def sync_index(self):
             self.generate_index()
             return
         if len(index_paths) > 20:
-            warnings.warn(f"The index basket count is {len(index_paths)}. " +
-                 "Consider running weave.IndexPandas.clean_up_indices")
+            if self.auto_cleanup:
+                self.clean_up_indices()
+            else:
```

Comment:
I think there's an issue with the text formatting here. When I test it out myself, I see the following: some extra text:
![image](https://github.com/user-attachments/assets/eb46cd33-edb9-44d7-8003-60a37a24c05f)

Reply:
I updated this so that the warning.warn does not look cut off by using a variable warn instead of the full string